### ISSUE TITLE
Issue with path slashes on Windows

### ISSUE THREAD

**Issue by lcharette** [userfrosting/UserFrosting#1133]
See test result : https://github.com/userfrosting/UserFrosting/runs/1622445785?check_suite_focus=true#step:7:1098

```
There was 1 failure:

1) UserFrosting\Sprinkle\Core\Tests\Integration\Filesystem\FilesystemTest::testAddingAdapter
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'D:/a/UserFrosting/UserFrosting/app/storage/testingDriver/'
+'D:/a/UserFrosting/UserFrosting/app/storage/testingDriver\'

D:\a\UserFrosting\UserFrosting\app\sprinkles\core\tests\Integration\Filesystem\FilesystemTest.php:150
```

This should be linked to the above : 
```
23) UserFrosting\Sprinkle\Core\Tests\Integration\I18n\SiteLocaleTest::testgetAvailableOptions
UserFrosting\Support\Exception\FileNotFoundException: The repository file 'locale://fr_FR/locale.yaml' could not be found.

D:\a\UserFrosting\UserFrosting\app\vendor\userfrosting\support\src\Repository\Loader\FileRepositoryLoader.php:81
D:\a\UserFrosting\UserFrosting\app\vendor\userfrosting\support\src\Repository\Loader\FileRepositoryLoader.php:58
D:\a\UserFrosting\UserFrosting\app\vendor\userfrosting\i18n\src\Locale.php:62
D:\a\UserFrosting\UserFrosting\app\vendor\userfrosting\i18n\src\Locale.php:51
D:\a\UserFrosting\UserFrosting\app\sprinkles\core\src\I18n\SiteLocale.php:47
D:\a\UserFrosting\UserFrosting\app\sprinkles\core\src\I18n\SiteLocale.php:76
D:\a\UserFrosting\UserFrosting\app\sprinkles\core\tests\Integration\I18n\SiteLocaleTest.php:101
```

@abdullahseba Had you found a fix for this ?

**Comment by lcharette** [comment]
So locale issue was fixed in 5313fe679df29ee9ef65ff3d883969236138b7ce. It was failing because of a conflict between absolute and relative path on Windows. A test has been added in i18n to replicate the issue : https://github.com/userfrosting/i18n/blob/be51a4c3e62567737cd598afed8a3e614b30e2a2/tests/LocaleTest.php#L247-L275

Which is now failing : https://github.com/userfrosting/i18n/runs/1623644778?check_suite_focus=true#step:5:18

**Comment by lcharette** [comment]
As for the path issue, [flysystem](https://github.com/thephpleague/flysystem/blob/13352d2303b67ecfc1306ef1fdb507df1a0fc79f/src/Adapter/Local.php#L47) Local adapter replace any trailing slashes with the `DIRECTORY_SEPARATOR` constant. At some point our `\UserFrosting\DS` should probably use that constant too, but this would maker a lot of mess all around UF code.
